### DISCUSSION TITLE
Welcome to shortyy Discussions!

### DISCUSSION THREAD

**Discussion by FireHead90544** [FireHead90544/shortyy#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Share your opinions and suggestions.
  * Help us capture annoying bugs.
  * Engage with other community members.
  * Ask QnAs.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by imadill** [comment]


**Reply by FireHead90544** [reply]
Done 👍
